utility/src/main/java/org/oppia/util/parser/LiTagHandler.kt

```diff
@@ -8,12 +8,12 @@ import android.text.Spanned
 import android.text.Spanned.SPAN_EXCLUSIVE_EXCLUSIVE
 import android.text.Spanned.SPAN_MARK_MARK
 import android.text.style.LeadingMarginSpan
-import java.util.Stack
 import org.oppia.util.parser.LiTagHandler.ListTag
 import org.oppia.util.parser.StringUtils.LI_TAG
 import org.oppia.util.parser.StringUtils.OL_TAG
 import org.oppia.util.parser.StringUtils.UL_TAG
 import org.xml.sax.XMLReader
+import java.util.*
```

Comment:
Please expand this per https://developer.android.com/kotlin/style-guide#import_statements.

Reply:
Done.